Cesium3DTilesSelection/include/Cesium3DTilesSelection/CreditSystem.h

```diff
@@ -91,5 +90,6 @@ class CESIUM3DTILESSELECTION_API CreditSystem final {
   int32_t _currentFrameNumber = 0;
   std::vector<Credit> _creditsToShowThisFrame;
   std::vector<Credit> _creditsToNoLongerShowThisFrame;
+  std::unordered_map<size_t, int32_t> _creditCounts;
```

Comment:
Can you store the count in `HtmlAndLastFrameNumber`? I think it would be quite a bit more efficient that way. Credits are worth a bit of optimization, because they're used extensively every frame. Cozzi was just making fun of me today for my initial implementation of credits in CesiumJS showing up as a hotspot in the profiler. 😆